keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#include "ergodox.h"
+// #include "debug.h"
+#include "action_layer.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here. 

Everything but the version.h and keymap_colemak.h.